keyboards/asmodus/plaquead/info.json

```
@@ -0,0 +1,2459 @@
+ {
+    "maintainer": "Mai The San",
+    "manufacturer": "Asmodus",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP3", "GP1", "GP0", "GP25", "GP11", "GP9"],
+        "cols": ["GP29", "GP27", "GP26", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP15", "GP14", "GP13", "GP5", "GP4"]
+    },
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "community_layouts": [
+        "tkl_ansi",
+        "tkl_ansi_split_bs_rshift",
+        "tkl_ansi_tsangan",
+        "tkl_ansi_tsangan_split_bs_rshift",
+        "tkl_iso",
+        "tkl_iso_split_bs_rshift",
+        "tkl_iso_tsangan",
+        "tkl_iso_tsangan_split_bs_rshift",
+        "tkl_f13_ansi",
+        "tkl_f13_ansi_split_bs_rshift",
+        "tkl_f13_ansi_tsangan",
+        "tkl_f13_ansi_tsangan_split_bs_rshift",
+        "tkl_f13_iso",
+        "tkl_f13_iso_split_bs_rshift",
+        "tkl_f13_iso_tsangan",
+        "tkl_f13_iso_tsangan_split_bs_rshift"
```

1. add tkl*wkl Community Layouts
2. sort Community Layouts literal
```suggestion
        "tkl_ansi_wkl",
        "tkl_ansi_wkl_split_bs_rshift",
        "tkl_f13_ansi",
        "tkl_f13_ansi_split_bs_rshift",
        "tkl_f13_ansi_tsangan",
        "tkl_f13_ansi_tsangan_split_bs_rshift",
        "tkl_f13_ansi_wkl",
        "tkl_f13_ansi_wkl_split_bs_rshift",
        "tkl_f13_iso",
        "tkl_f13_iso_split_bs_rshift",
        "tkl_f13_iso_tsangan",
        "tkl_f13_iso_tsangan_split_bs_rshift",
        "tkl_f13_iso_wkl",
        "tkl_f13_iso_wkl_split_bs_rshift",
        "tkl_iso",
        "tkl_iso_split_bs_rshift",
        "tkl_iso_tsangan",
        "tkl_iso_tsangan_split_bs_rshift",
        "tkl_iso_wkl",
        "tkl_iso_wkl_split_bs_rshift"
```